charts/jenkins/values.yaml

```diff
@@ -278,6 +278,10 @@ master:
   JCasC:
     enabled: true
     defaultConfig: true
+    # configScripts:
+    #   welcome-message: |
```

Comment:
Can we move this example below config scrips section. I'd love to avoid that people simply uncomment it and have two configScripts sections afterwards.

Reply:
Sorry, it looks like I've messed up commit history a bit. Could you take a look?